vue-gtag.d.ts

```diff
@@ -0,0 +1,78 @@
+declare module 'vue-gtag' {
+  import _Vue, { PluginFunction } from 'vue';
+  import VueRouter from 'vue-router';
+
+  export interface PageView {
+    page_path: string;
+  }
+
+  export interface Event {
```

Comment:
we are missing the action label. 
the way this is structured is slightly different from the pageview: there's first a string that describes the action and then the parameters inside are the one that you have added already.
you can see what I mean looking at this line https://github.com/MatteoGabriele/vue-gtag/blob/master/src/api/event.js#L4

Reply:
From your api and (https://developers.google.com/analytics/devguides/collection/gtagjs/events) it seems like action is on the event function in the interface I've called VueGtag.

I think that event has everything else, apologies if I'm misunderstanding something.

Reply:
Oh man I didn't see that part. Sorry about that!

I should rename it `action`  instead of `name` to be honest: it's a better name and is in sync with google's api, but not a priority by any means.